Builds/containers/packaging/rpm/build_rpm.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -ex
+
+source update_sources.sh
+
+# Build the rpm
+
+IFS='-' read -r RIPPLED_RPM_VERSION RELEASE <<< "$RIPPLED_VERSION"
+export RIPPLED_RPM_VERSION
+
+RPM_RELEASE=${RPM_RELEASE-1}
+
+# post-release version
+if [ "hf" = "$(echo "$RELEASE" | cut -c -2)" ]; then
+    RPM_RELEASE="${RPM_RELEASE}.${RELEASE}"
+# pre-release version (-b or -rc)
+elif [[ $RELEASE ]]; then
+    RPM_RELEASE="0.${RPM_RELEASE}.${RELEASE}"
```

Comment:
Am I reading this right that 1.3.0-b1 becomes 0.1.3.0.b1?

Reply:
not exactly - more like `rippled-1.2.0-0.1.rc2.el7.x86_64.rpm`. This is the naming scheme we have been using for a few releases, and although it's likely not perfect, I'd rather not change it in the context of this commit. I'll look into future improvements based on https://fedoraproject.org/wiki/Packaging:Versioning